### DISCUSSION TITLE
Sort Soundtrack under Albumartist

### DISCUSSION THREAD

**Discussion by alexe0506** [beetbox/beets#5118] [Q&A]
I want beets to sort every album tagged as soundtrack under their own Albumartist path. As an example i want Prince - Purple Rain to be placed within the Prince folder and not a new soundtrack folder. I have tried the code under with no success. Any help would be greatly appreciated.
`albumtype:soundtrack: $albumartist/[$year] $albumartist - $album ($original_year, $media, $label $catalognum, $format) /%if{$isMultidisc,Disc [$disc] $disctitle/}$track - $title`

**Comment by wisp3rwind** [comment]
I think we'll need more information to be able to help here: What's your full configuration? There may be other path configs taking precedence. Can you check (e.g. using `beet ls`  or `beet info`) that the albums in questions do in fact have `albumtype=soundtrack`?
